.github/scripts/generate_ci_workflows.py

```diff
@@ -49,6 +49,7 @@
 class CIFlowConfig:
     enabled: bool = False
     labels: Set[str] = field(default_factory=set)
+    trigger_action_only: bool = False
```

Comment:
Just the clarify, does trigger_action_only being True mean that the workflow will only run if the trigger action is enabled and not for any other case?

Reply:
```
    {%- if ciflow_config.trigger_action_only %}
    types: [!{{ ciflow_config.trigger_action }}]
    {%- else %}
    types: [opened, synchronize, reopened, !{{ ciflow_config.trigger_action }}]
    {%- endif %}
```

It controls if we listen on 
- types: `[opened, synchronize, reopened, !{{ ciflow_config.trigger_action }}]` or just 
- types: `[!{{ ciflow_config.trigger_action }}]`

Going to add some comments...